README.md

```diff
@@ -59,6 +59,16 @@ zmqpubrawblock=tcp://127.0.0.1:29000
 zmqpubrawtx=tcp://127.0.0.1:29000
 ```
 
+Depending on the actual hardware configuration, it may be useful to provide increased `dbcache` parameter value for faster verification and `rpcthreads` parameter value for better handling of multiple simultaneous API requests from `bitcoind` side. 
+
+```conf
+dbcache=2048
+rpcworkqueue=128
+rpcthreads=128
```

Comment:
I really wouldn't recommend that. If you want the best performance, you shouldn't have more threads than your number of virtual CPUs. If you put that setting on a hardware with a small number of vCPUs, you'll be able to keep more requests pending in memory, but you'll strongly degrade your performance (context-switching all the time can be expensive and is unnecessary here) so you'll take a lot more time handling these requests.

Raising `rpcworkqueue` is a much better choice here, it gives you the benefits you want without the drawbacks.

Reply:
> I really wouldn't recommend that. If you want the best performance, you shouldn't have more threads than your number of virtual CPUs. If you put that setting on a hardware with a small number of vCPUs, you'll be able to keep more requests pending in memory, but you'll strongly degrade your performance (context-switching all the time can be expensive and is unnecessary here) so you'll take a lot more time handling these requests.

You are likely get it wrong `rpcthreads` is not related to CPU threads. It is about RPC threads in a sense of accepted requests. Default is 4 I'm testing some more on various hardware setups. No results yet.

Reply:
Interesting, I didn't know that! I'll check the bitcoin code in more details to better understand that parameter.

Reply:
Are you sure about that? I'm looking at https://github.com/bitcoin/bitcoin/blob/b784ab10f555b043c77a2356faff492c35a3404c/src/httpserver.cpp#L417, I'm not 100% sure but it really looks like these are `std::thread` objects, which are OS threads AFAIK?

Reply:
Let maybe keep it around 4? I do not see clear recommendations from documentation.
https://en.cppreference.com/w/cpp/thread/thread

Reply:
I think we should remove it from the recommendations in that case, it's the same default value as what bitcoin core sets.

Reply:
I just tried 128. I can't report about meaningful difference also because my node is not under heavy load. I'd suggested to leave it but you are free to remove it of course.

Reply:
I don't think we should keep a parameter for which we don't have a good recommendation, can you please remove it and we should be good to merge?
And don't forget to remove it from the line above as well (where you can instead mention `rpcworkqueue`.

Reply:
Done